lua/org-bullets.lua

```diff
@@ -18,12 +18,13 @@ local list_groups = {
 local defaults = {
   show_current_line = false,
   symbols = {
-    headlines = { "◉", "○", "✸", "✿" },
+    headlines = { " ", " ", " " },
     checkboxes = {
-      cancelled = { "", "OrgCancelled" },
-      done = { "✓", "OrgDone" },
-      todo = { "˟", "OrgTODO" },
+      half = { "", "OrgTSCheckboxHalfChecked" },
```

Comment:
I would prefer some sort of clock here as a clear sign than the empty circle since that doesn't immediately convey anything, but it's not a blocking change I might tweak it later